cmd/skopeo/utils.go

```diff
@@ -202,6 +202,21 @@ func (opts *imageDestOptions) newSystemContext() (*types.SystemContext, error) {
 	return ctx, err
 }
 
+func (opts *globalOptions) newSystemContext() (*types.SystemContext, error) {
```

Comment:
- Due to the security impact, this should have tests
- `imageOptions.newSystemContext()` should call this to ensure the two don’t drift apart from each other.